opa.c

```diff
@@ -333,15 +334,28 @@ opa_socket_context parse_opa_socket_eval_result(char *json)
         pr_info("nasp: opa result policy_prio [%d], egress_prio [%d]\n", policy_prio, egress_prio);
         pr_info("nasp: matched policy %s", json_serialize_to_string(json_object_get_wrapping_value(matched_policy)));
 
+        const char *ttl = json_object_dotget_string(matched_policy, "egress.properties.ttl");
+        size_t ttl_len = json_object_dotget_string_len(matched_policy, "egress.properties.ttl");
```

Comment:
I sense some error here. I saw that if the "egress.properties.ttl "object is empty this will return 0. But in line 341 if that value is NULL we are overwriting it with a value from a different object.  Shouldn't we update the len value as well? It could happen that the len is 0 but the TTL is not NULL.

Reply:
Good catch! Fixed.